Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -131,11 +131,24 @@ private string ConvertJToken(JToken jToken)
                         string _ => $"\"{value}\"",
                         _ => value?.ToString()?.Replace(',', '.'), // Replace decimal commas.
                     };
+
                 case JArray jArray:
-                    return $"new[] {{ {string.Join(", ", jArray.Select(ConvertJToken))} }}";
+                    var token = string.Join(", ", jArray.Select(ConvertJToken));
+                    var format = token.Contains("ListValueOption") ? "\n\t\t\t" : "";

```

Comment:
The indentation used in other parts of the generated code is spaces, not tabs. I don't want to debate tabs :) but with this the indentation will be off with rest of the code. Also see the tabs below.

Reply:
Are you saying you want me to replace the tabs with spaces? That is definitely not a problem. I did it strictly for readability.

Reply:
Yes, please.